src/static/style.css

```diff
@@ -454,6 +454,10 @@ main .boxes.center {
 .wiki .entry h4 {
   font-size: 18px;
   margin-bottom: 0;
+
+}
+.wiki .entry h4.bullet::before {
+  content: "▸ ";
```

Comment:
I would prefer the normal h4 => remove
Absolutely, certainly not the triangle in front, as that usually indicates some is / can be expanded.